rediscluster/client.py

```diff
@@ -131,7 +146,7 @@ class StrictRedisCluster(StrictRedis):
 
     def __init__(self, host=None, port=None, startup_nodes=None, max_connections=None, max_connections_per_node=False, init_slot_cache=True,
                  readonly_mode=False, reinitialize_steps=None, skip_full_coverage_check=False, nodemanager_follow_cluster=False,
-                 connection_class=None, **kwargs):
+                 connection_class=None, enable_read_from_replicas=False, **kwargs):
```

Comment:
I am not super into having `enable` inside the variable name. `read_from_replicas=False` is good enough.